### DISCUSSION TITLE
import js fancybox with laravel via npm

### DISCUSSION THREAD

**Discussion by InnovacionesSambel** [fancyapps/ui#466] [Q&A]
Hello, I have a project in Laravel 7 and I already installed fancybox through npm as indicated in the documentation "npm install --save @fancyapps/ui", I already imported the css, but when I wanted to import the js it doesn't recognize them, they could help me help please thank you

In the resources/js/app.js file I have tried to import it as follows:

```
require('@fancyapps/ui/dist/fancybox/fancybox.esm.js');
require('@fancyapps/ui/dist/fancybox/fancybox.umd.js');
```

```
require('@fancyapps/ui/dist/fancybox/fancybox.esm');
require('@fancyapps/ui/dist/fancybox/fancybox.umd');
```

with none of the above options it works for me.

In the resources/js/bootstrap.js file I have placed the following, but it doesn't work for me either

`window.fancyapps = fancyapps;`

**Comment by fancyapps** [comment]
Hi,

Sorry, I am not Laravel expert. I think you should simply follow the appropriate Laravel documentation (https://laravel.com/docs/7.x/mix) on how to include assets in the frontend. Maybe that helps,too -
https://laracasts.com/discuss/channels/javascript/include-npm-package-in-laravel-project?page=1&replyId=441822
